src/EnergyPlus/ZoneTempPredictorCorrector.cc

```diff
@@ -2920,17 +2922,16 @@ namespace ZoneTempPredictorCorrector {
                                     Zone(Loop).Name);
                 SetupOutputVariable("Zone Predicted Moisture Load to Humidifying Setpoint Moisture Transfer Rate",
                                     OutputProcessor::Unit::kgWater_s,
-                                    ZoneSysMoistureDemand(Loop).OutputRequiredToHumidifyingSP,
+                                    MoisturePredictedHumSPRate(Loop),
                                     "System",
                                     "Average",
                                     Zone(Loop).Name);
                 SetupOutputVariable("Zone Predicted Moisture Load to Dehumidifying Setpoint Moisture Transfer Rate",
                                     OutputProcessor::Unit::kgWater_s,
-                                    ZoneSysMoistureDemand(Loop).OutputRequiredToDehumidifyingSP,
+                                    MoisturePredictedDehumSPRate(Loop),
```

Comment:
@RKStrand I prefer to report both moisture loads with a single zone and with multipliers. The single zone load is predicted by the Predictor. The load with multipliers is used for a system. Therefore, we can give users a choice to select which output or both to be reported.

Reply:
@lgu1234 I can add new variables to this for moisture and sensible...hopefully this can still get in before the release.  Adding variables might delay it, but I understand the desire to have both single zone and multiplier quantities.